docs/flow/modin/backends/pandas/query_compiler.rst

```diff
@@ -0,0 +1,12 @@
+Pandas query compiler
+"""""""""""""""""""""
+`PandasQueryCompiler` is responsible for compiling efficient DataFrame algebra queries for the
```

Comment:
```suggestion
``PandasQueryCompiler`` is responsible for compiling a set of known user-defined functions
and pairing those with dataframe algebra operators in the
```

Reply:
What is "a set of known UDFs" and how does pandas query compiler responsible for compiling those?

Reply:
We use pandas functions as the UDFs usually, but sometimes we construct different functions. Since the backend does not assume anything about the functions they can be arbitrary. Maybe instead of user-defined functions we can use something like developer defined functions or predefined functions.

Reply:
changed to:
"``PandasQueryCompiler`` is responsible for compiling a set of known **predefined** functions and pairing those with dataframe algebra operators"